src/wp-includes/option.php

```diff
@@ -1283,9 +1283,9 @@ function delete_option( $option ) {
  * @since 6.6.0
  * @access private
  *
- * @param string $option          The name of the option.
- * @param mixed $value            The value of the option to check its autoload value.
- * @param mixed $serialized_value The serialized value of the option to check its autoload value.
+ * @param string    $option          The name of the option.
+ * @param mixed     $value            The value of the option to check its autoload value.
+ * @param mixed     $serialized_value The serialized value of the option to check its autoload value.
  * @param bool|null $autoload     The autoload value to check.
```

Comment:
```suggestion
 * @param mixed     $serialized_value The serialized value of the option to check its autoload value.
 * @param bool|null $autoload         The autoload value to check.
```

Alignments

Reply:
Hey @mukeshpanchal27, thanks for the review. I have fixed the alignment issue; please check again.